examples/saved_filters_example_test.go

```diff
@@ -0,0 +1,92 @@
+//+build examples
+
+package examples
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSavedFiltersAPIExample(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(runCRUDSavedFilters())
+	// assert.NoError(runGetAllSavedFilters())
+}
+
+func runGetAllSavedFilters() error {
+	client := NewClient()
+
+	fmt.Println("### GET ALL")
+	savedFilters, err := client.SavedFilters.GetAll(context.Background())
+	if err != nil {
+		return err
+	}
+	PrettyPrint(savedFilters)
+	fmt.Println()
+
+	return nil
+}
+
+func runCRUDSavedFilters() error {
+	client := NewClient()
+
+	fmt.Println("### GET")
+	savedFilter, err := client.SavedFilters.Get(context.Background(), 8275)
```

Comment:
May be better to call GET after CREATE, with the created filter ID. So the ID always exists.